cashctrl_ledger/ledger.py

```diff
@@ -77,11 +221,21 @@ def delete_price():
         """
         raise NotImplementedError
 
-    def delete_vat_code():
+    def delete_vat_code(self, code: str):
         """
-        Not implemented yet
+        Deletes a VAT code from the remote CashCtrl account.
+
+        Parameters:
+        ----------
+        code : str
+            The VAT code name to be deleted.
         """
-        raise NotImplementedError
+
+        remote_vats = self._client.list_tax_rates()
+        remote_vat_to_delete = remote_vats.loc[remote_vats['name'] == code]
+        delete_ids = ",".join(remote_vat_to_delete['id'].astype(str)) if not remote_vat_to_delete.empty else None
+
+        self._client.post('tax/delete.json', {'ids': delete_ids})
```

Comment:
Suggestion for improved readability:
- For if statements, use code blocks instead of one-line expressions
```
def delete_vat_code(self, code: str):
    remote_vats = self._client.list_tax_rates()
    to_delete = remote_vats.loc[remote_vats['name'] == code]
    if not to_delete.empty:
        delete_ids = ",".join(to_delete['id'].astype(str))
        self._client.post('tax/delete.json', {'ids': delete_ids})
```